bullet-featherstone/src/Base.hh

```diff
@@ -99,7 +101,7 @@ struct ModelInfo
     std::string _name,
     Identity _world,
     Eigen::Isometry3d _baseInertiaToLinkFrame,
-    std::unique_ptr<btMultiBody> _body)
+    std::shared_ptr<btMultiBody> _body)
```

Comment:
Why was this ownership change (here and elsewhere) necessary?  Is a unique_ptr not still sufficient here?

Reply:
yeah, all we now use a single `btMultiBody` to store links from both the top level model and also nested models. So the nested model now keeps a shared pointer to it.